functions/package.json

```diff
@@ -3,14 +3,15 @@
   "scripts": {
     "build": "tsc",
     "serve": "npm run build && firebase emulators:start --only functions",
-    "shell": "npm run build && firebase functions:shell",
+    "shell": "npm run config && npm run build && firebase functions:shell",
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log",
     "lint": "gts lint",
     "clean": "gts clean",
     "compile": "tsc",
-    "fix": "gts fix"
+    "fix": "gts fix",
+    "config": "firebase functions:config:get > .runtimeconfig.json"
```

Comment:
Is our Github Action going to need to be modified to support the new configuration?

Reply:
This is only necessary for running the code locally. The PurpleAir API keys are already deployed, and the `firebase functions:config:get` command line function returns the deployed configuration. For some reason, the emulators cannot use the deployed configuration, so you have to create a runtime file to use them.